public/js/src/module/last-saved.js

```diff
@@ -0,0 +1,152 @@
+import encryptor from './encryptor';
```

Comment:
All of the last-saved implementation now lives here. These changes should _mostly_ look familiar, as it's very similar to the previous implementation. The primary differences are:

- Last-saved records are no longer stored in `surveys`, they're written to a new `lastSavedRecords` store
- Some code style changes:
    - Named exports declared at the definition site; this improves the ability to follow references through the project, as well as makes it clearer at definition site whether a function is intended to be used outside the module
    - Arrow functions (they're just more idiomatic)
- Not technically a change, but calling out that _if_ a survey is already cached for offline mode, online mode will still update that cache's last-saved `externalData` instances when a survey changes. This ensures caches do not store stale last-saved data, particularly in cases where a last-saved instance may have been removed from the form.